config/locales/en.yml

```diff
@@ -146,6 +146,7 @@ en:
   layout:
     support:
       title: Get support
+      support_days: Monday to Friday (except public holidays)
```

Comment:
@duncanjbrown I removed this because it was felt it was unnecessary given the SLA gives a response time that is working days. See [this thread on Slack](https://ukgovernmentdfe.slack.com/archives/CN9Q1VB2M/p1599115518014200?thread_ts=1598949188.002000&cid=CN9Q1VB2M).

Reply:
Oops — thanks! Will remove

Reply:
Gone